File: Wikipedia Maçkolik-TFF Match Scraper/NewFolder1/Match.cs
using System;

namespace TFFScraper
{
    public class Match
    {
        public int TFFId { get; set; }
        public string HomeId { get; set; }

        public string AwayId { get; set; }
        public string StadiumId { get; set; }
        public string StadiumName { get; set; }
        public string Referee { get; set; }
        public string Referee2 { get; set; }
        public string Referee3 { get; set; }
        public string Referee4 { get; set; }
        public string Referee5 { get; set; }
        public string Referee6 { get; set; }
        public string Referee7 { get; set; }
        public int HomeMS { get; set; }
        public int AwayMS { get; set; }
        public DateTime Date { get; set; }


    }
}

File: docs/js/models/Match.js
/**
 * Match Model - Represents match data from TFF
 */
class Match {
    constructor() {
        this.tffId = 0;
        this.homeId = '';
        this.awayId = '';
        this.homeName = '';
        this.awayName = '';
        this.stadiumId = '';
        this.stadiumName = '';
        this.referee = '';
        this.referee2 = '';
        this.referee3 = '';
        this.referee4 = '';
        this.referee5 = '';
        this.referee6 = '';
        this.referee7 = '';
        this.homeMS = 0;
        this.awayMS = 0;
        this.isScoreSet = false;
        this.date = new Date();
        this.hasKnownTime = false;
        this.isDefaultWin = false;
    }

    /**
     * Create Match from TFF data
     */
    static fromTFFData(data) {
        const match = new Match();
        Object.assign(match, data);
        return match;
    }
}

// Export for use in other modules
if (typeof window !== 'undefined') {
    window.Match = Match;
}
